xontrib/branch_expansion.py

```diff
@@ -32,7 +33,11 @@ def expand_braces(cmd, **_):
                 prefix + option + postfix for option in options.split(",")
             ]
             expanded_cmds.extend(expanded_parts)
+            requires_expansion = True
```

Comment:
```suggestion
            return " ".join(expanded_cmds)
```

Reply:
but that way i don't know if there are parts with both `{` and `}` so i can stop the expansion